pages/glossary/owned-by.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Owned by
+key: owned-by
+---
+
+An element A is owned by element B if element B is [included in the accessibility tree](#included-in-the-accessibility-tree) and:
+
+1. Both elements exist within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees), and element A is the first element within the tree, in [tree order](https://www.w3.org/TR/dom/#concept-tree-order), whose ID is included within the `aria-owns` attribute of element B, or
+2. Element B is the closest [shadow including ancestor](https://www.w3.org/TR/dom41/#concept-shadow-including-ancestor) of element A.
+
+Nodes that are not included in the accessibility tree cannot be owned by other elements.
```

Comment:
Which one takes precedence?

Reply:
And what happens when multiple elements own the same element?